internal/ethapi/api.go

```diff
@@ -688,8 +688,8 @@ func (api *BlockChainAPI) GetBalance(ctx context.Context, address common.Address
 	if state == nil || err != nil {
 		return nil, err
 	}
-	b := state.GetBalance(address).ToBig()
-	return (*hexutil.Big)(b), state.Error()
+	nativeBalance, sgtBalance := core.GetGasBalancesInBig(state, api.b.ChainConfig(), address)
```

Comment:
Should we use a flag to turn it on or off?  Or another idea is to use a flag with different port number for SGT balance RPC

Reply:
What about re-using `OptimismConfig.UseSoulGasToken` as the flag to enable this change?